ProjectSettings/ProjectSettings.asset

```diff
@@ -607,7 +611,7 @@ PlayerSettings:
       EnterpriseAuthentication: False
       HumanInterfaceDevice: False
       InputInjectionBrokered: False
-      InternetClient: False
+      InternetClient: True
```

Comment:
Interesting that this hadn't already been set!

Reply:
I thought so too!

Reply:
Ohh interesting, this must be why we've never noticed it ([here](https://msdn.microsoft.com/library/windows/apps/hh464936) under **Internet and public networks**): 

> The internetClientServer capability includes the access that the internetClient capability provides, so you don't need to specify internetClient when you specify internetClientServer.

Probably doesn't hurt to keep it set though!

Reply:
Ahh I see.